modules/core/src/scenegraph/gltf-instantiator/gltf-instantiator.js

```diff
@@ -115,67 +116,66 @@ export default class GLTFInstantiator {
     });
     model.setProps({attributes});
 
-    this.loadImage(0).then(img => model.setUniforms({ tex: new Texture2D(this.gl, { data: img }) }));
+    if (gltfPrimitive.material) {
+      this.configureMaterial(model, gltfPrimitive.material);
```

Comment:
Add TODO to reset to some defaults if no material? Or just use previously set material? Or log warning?